wd_demo.au3

```diff
@@ -193,6 +216,10 @@ Func RunDemo($idDebugging, $idBrowsers, $idUpdate, $idHeadless)
 	Next
 
 	_RunDemo_ErrorHander(True, $iError, @extended, $iWebDriver_PID, $sSession, $sDemoName)
+
+	If $sOutput = '_DebugOut' Then
+;~ 		WinClose('wd_demo - console log output')
+	EndIf
```

Comment:
What is your plan for this?

Reply:
I'm wondering how to close DebugOut window without closing script.

Reply:
Found this -- https://www.autoitscript.com/forum/topic/119506-_debugout-close-or-kill-report-log-window/

Reply:
Take a look: ['_DebugOut' GUIDelete](https://github.com/Danp2/au3WebDriver/pull/208/commits/bb4f9f1b860bbdad312018f496e610b150d79b15)